docs/src/errors.rst

```diff
@@ -328,11 +328,21 @@ API
     Returns the error message for the given error code.  Leaks a few bytes
     of memory when you call it with an unknown error code.
 
+.. c:function:: char* uv_strerror_r(int err, char* buf, size_t buflen)
+
+    Returns the error message for the given error code. The zero-terminated
+    message is stored the user-supplied buffer buf of at most buflen bytes.
```

Comment:
is stored **in** the user-supplied 
Also, I would use backticks for `buf` and `buflen`